package.json

```diff
@@ -89,20 +89,20 @@
   },
   "dependencies": {
     "classnames": "2.2.5",
-    "flatpickr": "3.0.7",
+    "flatpickr": "4.0.7",
     "lodash.debounce": "^4.0.8",
     "react-addons-css-transition-group": "^15.4.2",
     "warning": "3.0.0",
     "window-or-global": "^1.0.1"
   },
   "devDependencies": {
     "@storybook/addon-actions": "^3.2.15",
-    "@storybook/addon-centered": "3.2.13",
+    "@storybook/addon-centered": "3.2.15",
```

Comment:
Are all the major changes in this diff okay? Definitely trust you if they don't break anything, but seeing some of these make me nervous.

Reply:
@joshblack Thank you for asking. The storybook stuff and test (local and Travis) had no problem with this change. Not for release stuffs, though - Let me know if you want to back those out for now.

Reply:
Only big one is just `flatpickr` for consumers so as long as that works seems like we'd be good 😄 

Maybe we could help protect against some of these by publishing this as a breaking change, just in case?

Reply:
Sounds good @joshblack - How does this project mark a change as breaking?